tests/e2e/package.json

```diff
@@ -16,6 +16,7 @@
   "author": "Ihor Okhrimenko (iokhrime@redhat.com)",
   "license": "ISC",
   "devDependencies": {
+    "@eclipse-che/che-devworkspace-generator": "0.0.1-602dcb4",
```

Comment:
So that it's PR into the `main` branch with tests of Eclipse Che Next, "che-devworkspace-generator" of next version is better to use in here:
```
"@eclipse-che/che-devworkspace-generator": "next",
```

Reply:
fixed